docs/source/django_rest_integration.rst

```diff
@@ -0,0 +1,45 @@
+Using with Django Rest Framwork
+=================================
+
+1. Add ``'django_multitenant.middleware.MultitenantMiddleware'`` to the ``MIDDLEWARE`` list in your ``settings.py`` file:
+
+    .. code-block:: python
+
+        MIDDLEWARE = [
+            # other middleware
+            'django_multitenant.middleware.MultitenantMiddleware',
+        ]
+
+2. Monkey patch ``django_multitenant.views.get_tenant`` function with your own function which returns tenant object:
+
+    .. code-block:: python
+
+        # views.py
+
+        def tenant_func(request):
+            return Store.objects.filter(user=request.user).first()
+
+        # Monkey patching get_tenant function
+        from django_multitenant import views
+        views.get_tenant = tenant_func
+3. Add your viewset derived from TenantViewSet:
```

Comment:
The article says `TenantViewSet` but the code says `TenantModelViewSet`

Reply:
I will fix the article.